src/components/LeafletMap.js

```diff
@@ -281,9 +256,7 @@ class LeafletMap extends Component {
         fullMapZoomHandler(false);
         this.map.closePopup();
         getMapBounds(bbox);
-      } else {
-        this.map.setView(mapProperty.center, mapProperty.zoom);
-      }
+      } else this.map.setView(mapProperty.center, mapProperty.zoom);
```

Comment:
This is purely stylistic, but I find the if the `if` has the curly-braces, I tend to prefer mirroring that in the else as well just for the sake of readability. 

No big deal either way.